packages/client/helpers/build.js

```diff
@@ -28,6 +30,22 @@ async function build() {
       target: ['chrome58', 'firefox57', 'safari11', 'edge16'],
       format: 'cjs',
     }),
+    // copy wasm files, etc necessary for undici
+    fs.copy(
+      path.resolve(
+        __dirname,
+        '../node_modules/@prisma/engine-core/node_modules/undici/lib/llhttp',
+      ),
+      path.resolve(__dirname, '../generator-build/llhttp'),
+    ),
+    // copy wasm files, etc necessary for undici
+    fs.copy(
+      path.resolve(
+        __dirname,
+        '../node_modules/@prisma/engine-core/node_modules/undici/lib/llhttp',
+      ),
+      path.resolve(__dirname, '../runtime/llhttp'),
+    ),
```

Comment:
why twice? why both these paths?

Reply:
Somehow the `BinaryEngine` gets bundled into `generator-build`, but does not seem to be used?
So is the copying if `lhttp` really a pre-requisite? Would it throw an error if not found?

Reply:
```suggestion
    // copy wasm files, etc necessary for undici
    fs.copy(
      path.resolve(
        __dirname,
        '../node_modules/@prisma/engine-core/node_modules/undici/lib/llhttp',
      ),
      path.resolve(__dirname, '../runtime/llhttp'),
    ),
```

Reply:
Let's see: https://github.com/prisma/prisma/pull/8889

Reply:
Seems the `generator-build` one was indeed not necessary (at least that is what the PR said), so merged removal of this into here. Should trigger a new integration release.